packages/components/src/components/notification/_inline-notification.scss

```diff
@@ -135,6 +135,32 @@
     word-break: break-word;
   }
 
+  .#{$prefix}--inline-notification__action-button.#{$prefix}--btn--sm {
```

Comment:
Do we need the extra small specificity since these are always small? Can we instead fix the small buttons (which should be 32px) and then this button can just have the color overrides?

Reply:
As of right now, it is required to work correctly. I'm waiting for another reviewer to give me the go-ahead to fix something outside of the scope of this PR. The small ghost button issue has been identified but not reported, so I'm working only with what is the reality fo the code.